skills/declarative/calendarSkill/dialogs/SearchEventsDialog/SearchEventsDialog.dialog

```diff
@@ -0,0 +1,373 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "id": "tMMCfJ",
+    "name": "SearchEventsDialog"
+  },
+  "autoEndDialog": true,
+  "defaultResultProperty": "dialog.result",
+  "triggers": [
+    {
+      "$kind": "Microsoft.OnBeginDialog",
+      "$designer": {
+        "name": "BeginDialog",
+        "id": "UnSYCl"
+      },
+      "actions": [
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "SapBYc"
+          },
+          "name": "SearchEventIn"
+        },
+        {
+          "$kind": "Microsoft.EmitEvent",
+          "$designer": {
+            "id": "ET8AYy"
+          },
+          "eventName": "CollectOptions"
+        },
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "wYIwoN"
+          }
+        },
+        {
+          "$kind": "Microsoft.IfCondition",
+          "$designer": {
+            "id": "ekNrAH"
+          },
+          "condition": "dialog.start == null && dialog.title == null && (dialog.personNameArray == null || count(dialog.personNameArray) == 0)",
+          "actions": [
+            {
+              "$kind": "Microsoft.TraceActivity",
+              "$designer": {
+                "id": "1eO2VL"
+              },
+              "name": "CollectOption"
+            },
+            {
+              "$kind": "Microsoft.DeleteProperties",
```

Comment:
You should be able to remove this step by setting "Always prompt" to true on the text input

Reply:
Fixed